src/manifold/src/csg_tree.cpp

```diff
@@ -161,6 +164,9 @@ Manifold::Impl CsgLeafNode::Compose(
     if (!glm::isfinite(nodePrecision)) nodePrecision = -1;
     precision = glm::max(precision, nodePrecision);
 
+    vertIndices.push_back(numVert);
+    edgeIndices.push_back(numEdge * 2);
+    triIndices.push_back(numTri);
```

Comment:
So this is the loop that segfaults when parallelized? I assume something is going wrong with the precision calculation? Does feel like a bug - I can't think of an immediate reason it couldn't be parallelized...